fixtures/replicationController.js

```diff
@@ -1,10 +1,10 @@
 window.EXAMPLE_RC = {
   "kind": "ReplicationController",
-  "apiVersion": "v1beta3",
+  "apiVersion": "v1",
```

Comment:
add `serviceAccountName` to the rc spec.template and to the replication controller view